simulation_ws/src/test_nodes/launch/docking_test.launch

```diff
@@ -0,0 +1,45 @@
+<launch>
+    <arg name="use_sim_time" default="true"/>
+
+    <param name="use_sim_time" value="$(arg use_sim_time)"/>
+
+    <!-- Enable or disable listening to a boolean topic for the test. If a "true" message is published,
+    the test will succeed.
+    [DEFAULT]: True -->
+    <param name="enable_is_docked_topic" type="bool" value="$(optenv ENABLE_IS_DOCKED_TOPIC true)" />
+    <!-- Name of the boolean topic name from which the test listens for docking completion
+    [DEFAULT]: "/is_docked"-->
+    <param name="docked_topic_name" type="bool" value="$(optenv DOCKED_TOPIC_NAME /is_docked)" />
```

Comment:
Now that I think, we can add to the previous `doc` that the topic by default is `/docked` and that should be remapped if needed.
Then, we remap the `/is_docked` to `/docked` so it's shown as an example and this parameter can be removed.